billy/utils/audit_events.py

```
@@ -0,0 +1,69 @@
+def string_attr(the_object, the_attribute):
+    prop = getattr(the_object, the_attribute, None)
+    if prop:
+        try:
+            return str(prop)
+        except:
```

wow, are you seriously swallowing a naked exception?